app/models/user.rb

```diff
@@ -514,8 +514,14 @@ def active_practice
     active_practices.first.id
   end
 
-  def follow(other_user)
+  def follow(other_user, watch)
```

Comment:
【MUST】
これだけだと第二引数が何の属性なのか呼び出し側から分からない（特に[テストコード](https://github.com/fjordllc/bootcamp/pull/3227/files#diff-91bef9fda3c55514e23a4a8edba948f68aad59b57951adcff0b11e9ea9159854R290)）ので以下のようにキーワード引数を使用してほしいです

```suggestion
  def follow(other_user, watch:)
```

呼び出し側からはこのように記述します

```ruby
current_user.follow(user, watch: watch)
```

Reply:
確かにそうですね！
`User#follow`にキーワード引数`watch`を設定しました。